src/scss/themes/_hero.scss

```diff
@@ -110,6 +116,12 @@
 			pointer-events: auto;
 		}
 
+		.o-teaser__heading {
```

Comment:
Adding `.o-teaser__heading a:visited` to the selectors would output less CSS and give us fewer places to update colours. Would you mind updating to this style? -- sorry to be pedantic! 

E.g.

```
 		.o-teaser__heading,
                .o-teaser__heading a:visited,
 		.o-teaser__meta,
 		.o-teaser__standfirst,
 		.o-teaser__timestamp {
 			color: oColorsGetPaletteColor('white');
  			pointer-events: auto;
  		}
```

rather than

```
 		.o-teaser__heading,
 		.o-teaser__meta,
 		.o-teaser__standfirst,
 		.o-teaser__timestamp {
 			color: oColorsGetPaletteColor('white');
  			pointer-events: auto;
  		}

		a:visited {
 			color: oColorsGetPaletteColor('white');
 		}
```

Reply:
@notlee Thank you for telling! I tidied up them.